Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM quay.io/3scale/apicast:master
```

Comment:
Haven't realised that earlier but using docker file we won't be able to use the image stream produced by open shift.